### DISCUSSION TITLE
Completely random aliases

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1480] [Feature Requests]
Hello!
I propose to add the function of generating completely random SimpleLogin addresses, consisting exclusively of a set of random numbers and letters. This should significantly increase user anonymity

**Comment by 4b15ed24f576** [comment]
Hi there! You can create completely random aliases directly from your dashboard.

You can also edit the generation rules in **Settings** > **Aliases**. If you select the option **"Based on UUID"**, each alias you will create will be a random string of characters.

![Yeb3h4B9dODMA0ge](https://user-images.githubusercontent.com/116150138/206707731-3674e23e-ddde-452a-9a87-04fdec854ff4.png)

![3Lz0fHfHPR6Py2V1](https://user-images.githubusercontent.com/116150138/206707736-7e362143-96f2-4c9f-9874-bc1e73a13239.png)

**Reply by ghost** [reply]
No-no. It is misunderstanding. I mean something like DuckDuckGo or AnonAddy do.

For example:
4iv6fox7f[@]simpelogin.com

**Reply by never-failing** [reply]
I support. Add a mask for generation please...
